tests/io/vasp/test_inputs.py

```diff
@@ -1293,6 +1293,11 @@ def test_automatic_monkhorst_vs_gamma_style_selection(self):
                     kpoints = Kpoints.automatic_density_by_lengths(struct, [len_density] * 3)
                     assert kpoints.style == Kpoints.supported_modes.Gamma
 
+    def test_non_ascii_comment(self):
+        """Non-ASCII comment like 'Γ' might not be encoded correctly in Windows."""
+        kpoints = Kpoints.from_file(f"{VASP_IN_DIR}/KPOINTS_band")
+        assert kpoints.labels[0] == "Γ"
```

Comment:
This should be able to verify the encoding, in my Windows machine (system language English but install Chinese input, so the default encoding becomes `gb2312`).

If I revert the explicit encoding, I would get exact error as that issue:
```
    def test_non_ascii_comment(self):
        """Non-ASCII comment like 'Γ' might not be encoded correctly in Windows."""
        kpoints = Kpoints.from_file(f"{VASP_IN_DIR}/KPOINTS_band")
>       assert kpoints.labels[0] == "Γ", kpoints.labels[0]
E       AssertionError: 螕
```

Reply:
No I didn't remove it, the code diff shown by a comment would not be updated (i.e. I updated the err msg in 2608e8a2eb107e510b2545e10edf4bafd7a1fae1 after this comment so you wouldn't see that part)

Reply:
Ah oki good then 😃 GitHub is also bugging for me today. Nice then! 👍🏻